cli/src/commands/fix.rs

```diff
@@ -245,13 +259,27 @@ fn run_tool(
         .current_dir(workspace_root)
         .stdin(Stdio::piped())
         .stdout(Stdio::piped())
+        .stderr(Stdio::piped())
         .spawn()
         .or(Err(()))?;
     let mut stdin = child.stdin.take().unwrap();
+    let stderr = child.stderr.take().unwrap();
     let output = std::thread::scope(|s| {
         s.spawn(move || {
             stdin.write_all(old_content).ok();
         });
+        s.spawn(move || {
+            let reader = BufReader::new(stderr);
+            for line in reader.lines() {
+                let Ok(line) = line else {
+                    continue;
+                };
+                if let Ok(stderr) = &mut stderr_mutex.lock() {
+                    (*stderr).write_all(line.as_bytes()).ok();
+                    (*stderr).write_all(b"\n").ok();
+                }
+            }
+        });
```

Comment:
Maybe better to collect stderr outputs to `Vec<u8>`? I don't think the latency would matter. If we don't want to buffer everything, I think we can send per-file stderr output over the channel, and print them by the main thread.

Reply:
I thought about that but didn’t know if it would be okay to buffer the entire stderr steam of a fix tool invocation. If that’s not a concern, then I like that change because it avoids interleaving output entirely.

Reply:
I updated it to collect stderr from each process into a Vec<u8>, then acquire the mutex and write it all at once.